Script/Packages/Atomic/UI.json

```diff
@@ -22,7 +22,9 @@
 		    "getWidgetAt<T extends UIWidget>(x: number, y: number, include_children: boolean): T;"
 		],
 		"UIWidget": [
-		    "getWidget<T extends UIWidget>(id: string): T;"
+		    "getWidget<T extends UIWidget>(id: string): T;",
+		    "onEvent: (data:any) => void;",
```

Comment:
Sorry if wasn't clear, I think we know this is a UIWidgetEvent, so ```eventData:UIWidgetEvent``` should work here as a parameter?